src/ec/packages/ec-component-footer-core/footer-core.story.js

```diff
@@ -1,14 +1,109 @@
+/* eslint-disable no-param-reassign, no-shadow */
 import { storiesOf } from '@storybook/html';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
 import withCode from '@ecl-twig/storybook-addon-code';
+import { withKnobs, text } from '@storybook/addon-knobs';
 
-import sections from './demo/data';
+import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
+import specs from './demo/data';
 import footer from './ecl-footer-core.html.twig';
 import notes from './README.md';
 
+const data = [...specs.sections];
+// Labels for the buttons.
+const requiredGroupId = 'Mandatory elements';
+const optionalGroupId = 'Optional elements';
+// Prepare the knobs.
+const formatFooter = data => {
+  // Site name.
+  data[0].title.link = {
+    label: text(
+      'sections[0].title.link.label',
+      data[0].title.link.label,
+      requiredGroupId
+    ),
+    path: text(
+      'sections[0].title.link.path',
+      data[0].title.link.path,
+      requiredGroupId
+    ),
+    placeholder: text('optional elements', 'none', optionalGroupId),
```

Comment:
Strange? I think it's confusing, maybe better to have the tabs and nothing inside it?

Reply:
I agree, but the tabs would simply not appear in case there is only one group defined.
That's the reason why of that "fake" prop, but i don't like it either, just that i think it's important to show the "mandatory elements" tab, since this is not going to be a single use case we can try finding a better solution, nothing against that, for now i think i will change the text inside the field to be something like "no optional element in this story"